stryker/angular/README.md

```diff
@@ -0,0 +1,41 @@
+# Angular
+Stryker supports Angular projects using the Angular CLI starting from @angular/cli v6.1.0. Are you using an older version? Then there are some tips later in this document.
+
+## @angular/cli 6.1.0 and higher
+Recommended stryker packages: `npm i -D stryker-api stryker stryker-karma-runner stryker-typescript stryker-html-reporter`
+
+### Configuration
+After installing the recommended packages, create the `stryker.conf.js` file in your repository. 
+The configuration below contains a good starting point for Angular projects. 
+You may have to change some paths or config settings like the selected browsers. 
+We highly suggest using a headless browser when testing using stryker.
+
+Coverage analysis with [stryker-jasmine](http://npmjs.com/package/stryker-jasmine) is unfortunately not supported as of right now.
+```js
+module.exports = function (config) {
+  config.set({
+    mutate: [
+      'src/**/*.ts',
+      '!src/**/*.spec.ts',
+      '!src/test.ts',
+      '!src/environments/*.ts'
+    ],
+    mutator: 'typescript',
+    testRunner: 'karma',
+    karma: {
+      configFile: 'src/karma.conf.js',
+      project: 'angular-cli',
```

Comment:
To avoid confusion about the intended purpose of this variable, this could be renamed to projectType .

Reply:
Good point! This could also prevent the confusion @blabno had.